### PR TITLE
Improve readme

### PR DESCRIPTION
**Pull request by rhornung67**
# Summary

- This PR updates and improves the README file.

### CODE REVIEW

**Review by rchen20 — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -13,36 +13,38 @@
 [![Join the chat at https://gitter.im/llnl/raja](https://badges.gitter.im/llnl/raja.svg)](https://gitter.im/llnl/raja?utm_source=badge&utm_medium=badge&utm_campaign=pr-badge&utm_content=badge)
 [![Coverage](https://img.shields.io/codecov/c/github/LLNL/RAJA/develop.svg)](https://codecov.io/gh/LLNL/RAJA)
 
-RAJA is a collection of C++ software abstractions, being developed at
+RAJA is a library of C++ software abstractions, primarily developed at
 Lawrence Livermore National Laboratory (LLNL), that enable architecture
```
Comment (trws) [inline README.md:17]:
```suggestion
Lawrence Livermore National Laboratory (LLNL), that enables architecture
```

File: README.md [hunk 2]
```diff
@@ -13,36 +13,38 @@
...
+    such as OpenMP, CUDA, etc. directly.
 
-RAJA uses standard C++11 -- C++ is the predominant programming language in
-which many LLNL codes are written. RAJA is rooted in a perspective based on 
-substantial experience working on production mesh-based multiphysics 
-applications at LLNL. Another goal of RAJA is to enable application developers
-to adapt RAJA concepts and specialize them for different code implementation 
-patterns and C++ usage, since data structures and algorithms vary widely 
-across applications.
+RAJA targets portable, parallel loop execution by providing building blocks
```
Comment (trws) [inline README.md:25]:
```suggestion
RAJA offers portable, parallel loop execution by providing building blocks 
```

File: README.md [hunk 3]
```diff
@@ -13,36 +13,38 @@
...
-substantial experience working on production mesh-based multiphysics 
-applications at LLNL. Another goal of RAJA is to enable application developers
-to adapt RAJA concepts and specialize them for different code implementation 
-patterns and C++ usage, since data structures and algorithms vary widely 
-across applications.
+RAJA targets portable, parallel loop execution by providing building blocks 
+that extend the generally-accepted parallel for idiom. RAJA relies on standard 
+C++11 features, such as lambda expressions.
+
+RAJA is rooted in a perspective based on decades of experience working on
```
Comment (trws) [inline README.md:29]:
```suggestion
RAJA's design is rooted in decades of experience working on 
```

**Review by trws — COMMENTED** [review]
Looks really good to me.  I popped a few comments in with suggestions for some little grammar or wording preference things, but this looks like a nice rework.

File: README.md (hunk 1 above)
Comment (rhornung67) [reply README.md:17]: Done.

File: README.md (hunk 2 above)
Comment (rhornung67) [reply README.md:25]: Done.

File: README.md (hunk 3 above)
Comment (rhornung67) [reply README.md:29]: Done.

**Comment by rhornung67** [comment]
All reviewer comments have been addressed.

File: README.md [hunk 4]
```diff
@@ -13,36 +13,38 @@
...
+  * To enable application portability with manageable disruption to existing 
+    algorithms and programming styles
+  * To achieve performance comparable to using common programming models, 
+    such as OpenMP, CUDA, etc. directly.
+
+RAJA offers portable, parallel loop execution by providing building blocks 
+that extend the generally-accepted parallel for idiom. RAJA relies on standard 
+C++11 features, such as lambda expressions.
+
+RAJA's desing is rooted in decades of experience working on
```
Comment (mdavis36) [inline README.md:29]: design

File: README.md [hunk 5]
```diff
@@ -13,36 +13,38 @@
...
+specialize them for different code implementation patterns and C++ usage.
 
 RAJA shares goals and concepts found in
 other C++ portability abstraction approaches, such as
 [Kokkos](https://github.com/kokkos/kokkos)
 and [Thrust](https://developer.nvidia.com/thrust). 
-However, it includes concepts that are absent in other models and which are 
-fundamental to LLNL codes. 
+However, it includes concepts and capabilities that are absent in other models 
+and which are fundamental to applications we work with.
```
Comment (mdavis36) [inline README.md:40]: remove either 'and' or 'which' here

**Review by mdavis36 — CHANGES_REQUESTED** [review]

File: README.md (hunk 4 above)
Comment (rhornung67) [reply README.md]: Fixed. Thanks.

File: README.md (hunk 5 above)
Comment (rhornung67) [reply README.md]: Done.

**Review by mdavis36 — APPROVED** [review]
Looks good :+1:

**Review by MrBurmark — APPROVED** [review]

**Review by trws — APPROVED** [review]
LGTM.

**Review by artv3 — APPROVED** [review]

**Pull request merged**